### PR TITLE
feat(FX-3732): Add “Alerts” item in top dropdown menu

### PR DESCRIPTION
**Pull request by dimatretyak**
Type: **Feature**
Jira ticket: [FX-3732]

### Acceptance Criteria
* “Alerts” item should be displayed in top dropdown menu on desktop (see Figma)
* Changes should be hidden under feature flag

#### Dekstop
https://user-images.githubusercontent.com/3513494/153385655-9223ea1f-e31e-4377-8c45-2d182f449212.mp4

#### Mobile
https://user-images.githubusercontent.com/3513494/153385705-c7302cd0-bc1b-4938-a4e5-645e391bc5f2.mp4

[FX-3732]: https://artsyproduct.atlassian.net/browse/FX-3732?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export const ENABLE_QUERY_BATCHING: any = false
 export const ENABLE_SAVED_SEARCH: any = false
 export const ENABLE_SERVER_SIDE_CACHE: any = false
 export const ENABLE_WEB_CRAWLING: any = false
+export const ENABLE_YOUR_ALERTS_PAGE: any = false
```
Comment (gkartalis) [inline src/config.ts:60]: **question:** Is there any reason that we use explicit `any` as type everywhere in this file 🤔 it is a more of a force specific question

**Review by gkartalis — APPROVED** [review]
🚀

File: src/config.ts (hunk 1 above)
Comment (dimatretyak) [reply src/config.ts:60]: Good question. But unfortunately I won't be able to answer it 🤷‍♂️

Comment (gkartalis) [reply src/config.ts:60]: No worries, feel free to merge from my side

File: src/v2/Components/NavBar/Menus/NavBarUserMenu.tsx [hunk 2]
```diff
@@ -65,6 +66,16 @@ export const NavBarUserMenu: React.FC = () => {
         <ReceiptIcon mr={1} aria-hidden="true" /> Order History
       </NavBarMenuItemLink>
 
+      {getENV("ENABLE_YOUR_ALERTS_PAGE") && (
```
Comment (dblandin) [inline src/v2/Components/NavBar/Menus/NavBarUserMenu.tsx:69]: Nice! Might be worth documenting this `getENV` utility in our "shared configuration" doc at some point: https://github.com/artsy/force/blob/main/docs/env_configuration.md

**Review by dblandin — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**